solutions/kubernetes/cluster-functions.sh

```diff
@@ -191,12 +201,16 @@ function install_prerequisites(){
         # enable cgroupfs 
         sed -i '/config.yaml/s/config.yaml"/config.yaml --cgroup-driver=cgroupfs"/g' /usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf || throw $ConfigException
 
+        # enable unix socket
+        sed -i 's/fd:\/\//unix:\/\//g' /usr/lib/systemd/system/docker.service && systemctl daemon-reload
+
         # enable local docker registry.
         mkdir -p /etc/docker/
         jq -n '{"insecure-registries": $ARGS.positional}' --args "cortx-docker.colo.seagate.com" > /etc/docker/daemon.json || throw $Exception
         echo "Configured /etc/docker/daemon.json for local docker registry"
```

Comment:
I guess this echo can be removed in light of new one.

Reply:
@nitisdev Removed extra echo statement. Please review again.